fabtests/unit/mr_cache_evict.c

```diff
@@ -591,8 +721,15 @@ static void usage(void)
 		"allocation is returned. This can be used to verify the \n"
 		"underlying physical memory changes between MMAP, BRK, and \n"
 		"SBRK allocations. When running as non-root, the reported \n"
-		"physical address is always zero.");
+		"physical address is always zero.\n\n"
+		"This test can also be used to verify caching of device memory\n"
+		"regions. Currently, FI_HMEM_CUDA is supported. The env variable\n"
+		"FI_MR_CUDA_CACHE_MONITOR_ENABLED can be used to disable the\n"
+		"caching of FI_HMEM_CUDA regions. Reported physical address is\n"
+		"always zero for device memory. The FI_HMEM capability hint must\n"
+		"be set to run device memory tests.");
```

Comment:
"The FI_HMEM capability hint must be set to run device memory tests." can be removed.